mlflow/system_metrics/metrics/gpu_monitor.py

```diff
@@ -0,0 +1,48 @@
+"""Class for monitoring GPU stats."""
+
+import logging
+import sys
+
+from mlflow.system_metrics.metrics.base_metrics_monitor import BaseMetricsMonitor
+
+_logger = logging.getLogger(__name__)
+
+try:
+    import pynvml
+except ImportError:
+    _logger.warning(
+        "`GPUMonitor` requires pynvml package. To install, run `pip install pynvml`. Skip "
+        "logging GPU metrics."
+    )
```

Comment:
Does this always get printed out when this module is imported? That might annoy users who don't need system metric logging.

Reply:
good call, let me remove the warning